### PR TITLE
Remove rsk3 lib

### PR DESCRIPTION
**Pull request by cesariovlabs**
## Types of changes

- [ ] fix: bug fixes, e.g. fix Button color on DarkTheme.
- [ ] feat: new features, e.g. add Snackbar component.
- [ ] refactor: code refactor, e.g. new folder structure for components.
- [ ] docs: changes into documentation, e.g. add usage example for Button.
- [ ] test: adding or updating tests, eg unit, snapshot testing.
- [ ] chore: tooling changes, e.g. change circleci config.
- [ ] BREAKING CHANGE: for changes that break existing usage, e.g. change API of a component.

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Tests for the changes have been added (for bug fixes / features)

* **Other information**:

* **SCREENSHOTS**:

### CODE REVIEW

**Comment by lucachaco** [comment]
I was able to send a transaction but is now loading the transaction history per token

File: src/common/apiHelper.js [hunk 1]
```diff
@@ -15,8 +15,8 @@ const apiHelper = {
     try {
       const parseServer = ParseHelper.getServerUrl();
       const url = `${parseServer}/functions/getAbiByAddress`;
-      // using Rsk3.utils.toChecksumAddress without chainId to match the backend to get the erc20 token's info(symbol, abi)
-      const result = await axios.post(url, { address: Rsk3.utils.toChecksumAddress(address) }, { headers, timeout: 5000 });
+      // using Web3.utils.toChecksumAddress without chainId to match the backend to get the erc20 token's info(symbol, abi)
+      const result = await axios.post(url, { address: Web3.utils.toChecksumAddress(address) }, { headers, timeout: 5000 });
```
Comment (ilanolkies) [inline src/common/apiHelper.js:19]: For the record, https://github.com/rsksmart/rsk3.js/blob/master/packages/rsk3-utils/src/index.js#L248

File: src/common/common.js [hunk 2]
```diff
@@ -628,9 +628,9 @@ const common = {
   async isContractAddress(address, chainId) {
```
Comment (ilanolkies) [inline src/common/common.js:628]: This one seems to be used for showing transactions notifications like this one https://github.com/rsksmart/rwallet/blob/develop/src/pages/dapp/browser.js#L608

It worked, tx ongoing

image

simulator_screenshot_062C0F82-F75E-48EF-86DD-60C147098B66

File: src/common/transaction/rbtccoin.js [hunk 3]
```diff
@@ -62,8 +62,8 @@ export const createRawTransaction = async ({
   symbol, type, sender: from, receiver: to, value, memo, gasPrice, gas, contractAddress,
 }) => {
   const rskEndpoint = type === 'Mainnet' ? MAINNET.RSK_END_POINT : TESTNET.RSK_END_POINT;
-  const rsk3 = new Rsk3(rskEndpoint);
-  const nonce = await rsk3.getTransactionCount(from, 'pending');
+  const web3 = new Web3(rskEndpoint);
+  const nonce = await web3.getTransactionCount(from, 'pending');
```
Comment (ilanolkies) [inline src/common/transaction/rbtccoin.js:66]: Nonce working, transactions passing ok

File: src/common/transaction/rbtccoin.js [hunk 4]
```diff
@@ -108,8 +108,8 @@ export const getRawTransactionParam = ({
 };
 
 export const signTransaction = async (rawTransaction, privateKey) => {
-  const rsk3 = new Rsk3('https://public-node.testnet.rsk.co');
-  const accountInfo = await rsk3.accounts.privateKeyToAccount(privateKey);
+  const web3 = new Web3('https://public-node.testnet.rsk.co');
+  const accountInfo = await web3.accounts.privateKeyToAccount(privateKey);
```
Comment (ilanolkies) [inline src/common/transaction/rbtccoin.js:112]: Signatures ok too

File: src/pages/dapp/browser.js [hunk 5]
```diff
@@ -9,7 +9,7 @@ import {
 } from 'react-native';
 import RNFS from 'react-native-fs';
 import PropTypes from 'prop-types';
-import Rsk3 from '@rsksmart/rsk3';
+import Web3 from 'web3';
```
Comment (ilanolkies) [inline src/pages/dapp/browser.js:12]: dApp browser working fine here. I can register RNS domains

File: src/pages/wallet/add.custom.token.js [hunk 6]
```diff
@@ -5,7 +5,7 @@ import {
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import _ from 'lodash';
-import Rsk3 from '@rsksmart/rsk3';
+import Web3 from 'web3';
```
Comment (ilanolkies) [inline src/pages/wallet/add.custom.token.js:8]: Custom token feat working too

File: src/pages/wallet/readonly/add.js [hunk 7]
```diff
@@ -5,7 +5,7 @@ import {
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import AntDesign from 'react-native-vector-icons/AntDesign';
-import Rsk3 from '@rsksmart/rsk3';
+import Web3 from 'web3';
```
Comment (ilanolkies) [inline src/pages/wallet/readonly/add.js:8]: Read-only wallet ok

File: src/pages/wallet/wallet.connect/index.js [hunk 8]
```diff
@@ -6,9 +6,8 @@ import WalletConnect from '@walletconnect/client';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import _ from 'lodash';
-import Rsk3 from '@rsksmart/rsk3';
+import Web3 from 'web3';
```
Comment (ilanolkies) [inline src/pages/wallet/wallet.connect/index.js:9]: Wasn't able to test it yet. I'm trying an approach setting Wallet Connect URI on an input text.  We can enable this for dev mode only. I will do PR later

**Review by ilanolkies — APPROVED** [review]

**Pull request merged**